File: src/main/java/corp/soyummy/userfavorites/dto/UserFavoriteUpdateDto.java
package corp.soyummy.userfavorites.dto;

import lombok.Getter;
import lombok.Setter;
import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UserFavoriteUpdateDto {
    @NotNull(message = "RecipeId is required!")
    @Size(min = 1, message = "RecipeId must have at least one character.")
    private String recipeId;
}

File: src/main/kotlin/com/soyummy/userfavorites/dto/UserFavoriteUpdateDto.kt
package com.soyummy.userfavorites.dto

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class UserFavoriteUpdateDto(
    @field:NotBlank(message = "RecipeId is required!")
    @field:Size(min = 1, message = "RecipeId must have at least one character.")
    val recipeId: String
)